R/setOrRetrieve.R

```diff
@@ -0,0 +1,108 @@
+.internal <- list2env(list(
+  recomputedHashtab = hashtab(type = "address", NULL), # unclear if "address" is the correct choice here
+  lastRecomputed    = TRUE
+), parent = emptyenv())
+
+saveHashOfJaspObject <- function(x) {
+  .internal[["recomputedHashtab"]][[x]] <- 0L
+}
+
+setRecomputed <- function(x) {
+  .internal[["lastRecomputed"]] <- x
+}
+
+#' Test if a jaspObject was recomputed of not.
+#' @param x a jaspObject, or missing in which case the last created or retrieved jaspObject is considered.
+#'
+#' @details Note that this function only works when \code{\link{\%setOrRetrieve\%}} is used to set or retrieve a jaspObject.
+#'
+#' @export
```

Comment:
I wonder if `isRecomputed` and `%setOrRetrieve%` could be put under one help file. This makes it clearer that they are to be used in tandem, which is also shown in the example file.

Reply:
done